README.md

```diff
@@ -122,34 +182,100 @@ yum install mosquitto
 
 See [mosquitto official website](https://mosquitto.org/download/) for more information.
 
-### Build Edge
+KubeEdge has certificate based authentication/authorization between cloud and edge. Certificates can be generated using openssl. Please follow below steps is any other method of generation of certificates isn't available.
```

Comment:
Please follow below steps is any other method of generation of certificates isn't available ->
Please follow the steps below to generate certificates. 

we don't need to specify non-availability of other methods. it is upto the user.

Reply:
Done, Thanks